src/service_client/assisted_service_api.py

```diff
@@ -76,7 +76,14 @@ def refresh_api_key(config: Configuration) -> None:
             }
 
             log.info("Refreshing API key")
-            response = requests.post(os.environ.get("SSO_URL"), data=params)
+            try:
+                sso_url = os.environ["SSO_URL"]
+            except KeyError:
+                log.error("The environment variable SSO_URL is mandatory but was not supplied")
+                raise
+            else:
+                response = requests.post(sso_url, data=params)
+
```

Comment:
Can you extract this to another PR to keep things clear ?

Reply:
Actually I think it should be in this PR because the absence of this env var prohibits one from running the download logs flow